apps/docs/src/components/code/Live.tsx

```diff
@@ -21,6 +22,7 @@ const imports = {
     react: React,
     "@hitachivantara/uikit-react-core": HvCore,
     "@hitachivantara/uikit-react-icons": HvIcons,
+    "@hitachivantara/uikit-react-code-editor": HvCodeEditor,
```

Comment:
Do we really to LivePreview import the code-editor package? It's a **large** package with a **single** component! I don't see us using in other components either

Reply:
I'll remove the `live` feature from the `CodeEditor` docs.

Reply:
nevermind

Reply:
nevermind, we do